client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.js

```diff
@@ -172,7 +170,7 @@ const returnMatchingCombination = (rpaTask, allMatchingCombinations) => {
     return undefined;
   }
   if (allMatchingCombinations.length > 1) {
-    if (rpaTask === 'Open Application') {
+    if (rpaTask === 'Open Application' || rpaTask === 'Open Workbook') {
```

Comment:
Please avoid hard coding these values. A solution that iterates over the possible combinations that collects the duplicates would be nicer.